luaui/Widgets/cmd_bar_hotkeys.lua

```diff
@@ -50,7 +51,17 @@ local function reloadBindings()
 	reloadWidgetsBindings()
 end
 
+local function copyDefaultCustom()
+	if VFS.FileExists("bar_hotkeys_custom.lua") then return end
+
+	local fileToCopy = Spring.GetConfigString("KeybindingFile", "luaui/configs/bar_hotkeys.lua")
+	fileToCopy = (fileToCopy ~= "bar_hotkeys_custom.lua") and fileToCopy or "luaui/configs/bar_hotkeys.lua"
+
+	copyFileString("luaui/configs/bar_hotkeys.lua", "bar_hotkeys_custom.lua")
```

Comment:
```suggestion
	copyFileString(fileToCopy, "bar_hotkeys_custom.lua")
```
If I correctly understand what this variable is for?

Reply:
The file to copy to setup the custom file, when it's not present. So if for example you are using some preset that is not the default and you don't have a custom file in your root it will be copied.

This is a bit faulty in the sense that you need to remove your custom file for it to be refreshed (as we don't want to remove custom files tweaked by the user). It is not bugged, but it's a feature people often don't know about.

Reply:
The PR I have to move things out of the lua format back to uikeys addresses this, as setting up a copy of your current bindings is trivial (issuing `/keysave <filename>` on chat)

Reply:
I think my concern about this piece of code is much simpler than that? The variable `fileToCopy` is set up to have a value, but then that value is not used

```lua
copyFileString("luaui/configs/bar_hotkeys.lua",...)
```
even when `Spring.GetConfigString("KeybindingFile")` is on e.g. grid keys and it should instead
```lua
copyFileString("luaui/configs/bar_hotkeys_grid.lua",...)
```
---
`/keysave` is certainly a useful tip though, I ended up copying the grid keyfile from github!